usl_pipeline/cloud_functions/main_test.py

```diff
@@ -40,12 +48,14 @@ def test_build_feature_matrix_flood(mock_storage_client, mock_firestore_client,
             raster.write(tiff_array.astype(rasterio.uint8), 1)
         tiff_bytes = memfile.read()
 
+    polygons_bytes = str.encode("1\n5 0 1 1 0 0 0 0 1 1 0\n")
```

Comment:
you can write bytes strings in Python with
```python
polygons_bytes = b"1\n5 0 1 1 0 0 0 0 1 1 0\n"
```

Reply:
Done